lib/internal/test_runner/test.js

```diff
@@ -655,6 +655,14 @@ class Test extends AsyncResource {
       this.parent.processReadySubtestRange(false);
       this.parent.processPendingSubtests();
     } else if (!this.reported) {
+      if (!this.passed && failed === 0 && this.error) {
+        this.reporter.fail(this.nesting, kFilename, this.testNumber, this.name, {
```

Comment:
```suggestion
        this.reporter.fail(0, kFilename, NaN, 'root', {
```

Reply:
updated except changed the `'root'` to the filename